tests/basic/misc_test.go

```diff
@@ -1784,4 +1784,71 @@ var _ = Describe("{KubeClusterRestart}", func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
-})
\ No newline at end of file
+})
+
+var _ = Describe("{VerifyNoPxRestartDueToPxPodStop}", func() {
+	JustBeforeEach(func() {
+		var testrailID = 87311294
```

Comment:
This might not be the correct test case ID. I can not verify it because TestRail is down now.